tests/framework/integration/testing.go

```diff
@@ -84,6 +98,13 @@ func BeforeTest(t testutil.TB, opts ...TestOption) {
 		testutil.RegisterLeakDetection(t)
 	}
 
+	if options.failpoint != nil && len(options.failpoint.name) != 0 {
+		require.NoError(t, gofail.Enable(options.failpoint.name, options.failpoint.payload))
```

Comment:
What happens if gofail is not enabled? Could we skip the test here with a message that user should call `make gofail-enable`